scripts/install

```diff
@@ -164,12 +164,13 @@ fi
 #=================================================
 
 # Add dynamic logout URL to the config
+url_base64="$(echo -n "https://$domain$path" | base64)"
 exec_occ config:system:get logout_url >/dev/null 2>&1 \
   || echo "
 //-YunoHost-
 // set logout_url according to main domain
 \$main_domain = exec('cat /etc/yunohost/current_host');
-\$CONFIG['logout_url'] = 'https://'.\$main_domain.'/yunohost/sso/?action=logout';
+\$CONFIG['logout_url'] = 'https://'.\$main_domain.'/yunohost/sso/?action=logout&r=${url_base64}';
```

Comment:
I'm not sure to understand why the main domain is used here compared to just `$domain` on which the app is installed ...

Anyway, this will probably stop working on 12.x / Bookworm because portals will now be "independent" in terms of "main domain trees", so if Nextcloud is installed on a different "main domain" this ain't gonna work

Also the `/yunohost/sso/?action=logout` endpoint is probably going to be `/yunohost/sso/logout`

But anyway there's no way to make this future-proof i think, we'll just have to patch Nextcloud once we'll be here